src/components/TodoComponents/Todo.js

```diff
@@ -1,8 +1,15 @@
+//   _____     ____         
+//  |_   _|__ |  _ \  ___ _ 
+//    | |/ _ \| | | |/ _ (_)
+//    | | (_) | |_| | (_) | 
+//    |_|\___/|____/ \___(_)
+                         
+
 import React from 'react';
 import './Todo.css';
 
 function Todo(props) {
-    return <p onClick={props.completedToggle}>{props.task}</p>
+    return <p className={`card ${props.completed}`} onMouseOver={props.pointer} onClick={() => props.completedToggle(props.id)} >{props.task}</p>
```

Comment:
You should definitely break this up for readability (which [prettier](https://prettier.io) would do for you):
```javascript
return (
  <p
    className={`card ${props.completed}`}
    onMouseOver={props.pointer} 
    onClick={() => props.completedToggle(props.id)}
  >
    {props.task}
  </p>
);
```
and here's a version assuming you've used [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
return (
  <p
    className={`card ${completed}`}
    onMouseOver={pointer} 
    onClick={() => completedToggle(id)}
  >
    {task}
  </p>
);
```